src/Init/Data/Float.lean

```diff
@@ -26,6 +26,7 @@ opaque floatSpec : FloatSpec := {
   decLe := fun _ _ => inferInstanceAs (Decidable True)
 }
 
+/-- Native floating point type, corresponding to 64-bit `double` in C. -/
```

Comment:
```suggestion
/-- Native floating point type, corresponding to the IEEE 754 *binary64* format
(also known as `double` in C or `f64` in Rust). -/
```
Avoid defining things in terms of C directly, since those definitions are target-dependent.

Reply:
Apologies, I didn’t see this until after it had been merged.

However, I believe Lean compiles to C, which does not technically guarantee IEEE compliance.  Assuming no extra step by Lean to ensure compliance, I went with what I thought was more universally correct.

Reply:
I believe we want to guarantee IEEE compliance, independent of what C does. We do not generally use C to define the semantics of our operations, and if C has target dependent behavior then we would `#ifdef` our way around it in order to get the semantics we want. The C backend may not even be the main backend in the future, with work on LLVM progressing.

Reply:
Sounds good, I’ll send another PR in a couple hours.

Reply:
https://github.com/leanprover/lean4/pull/3157